src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -84,6 +87,45 @@ public void run() {
         }
     }
 
+    private void niceSyncthing() {
+        new Thread() {
+            public void run() {
+                Process niceST = null;
+                DataOutputStream niceSTOut = null;
```

Comment:
`niceSTOut` kinda sucks for a camelCase var name, should be `niceStOut` or better just `niceOut`.